zcash_primitives/src/transaction/components/transparent/builder.rs

```diff
@@ -40,6 +40,26 @@ impl fmt::Display for Error {
     }
 }
 
+pub trait TransparentInput {
```

Comment:
- Needs documentation (the trait and its methods).
- Document this addition in the changelog.